src/encoded/static/components/item-pages/ExperimentView.js

```diff
@@ -329,26 +300,29 @@ const RawFilesTableSection = React.memo(function RawFilesTableSection(props){
     }
 
     // Get all files which have expUUID in experiments.uuid[] excluding reference and processed files.
-    const requestHref = `/search/?type=File&experiments.uuid=${encodeURIComponent(expUUID)}&type%21=FileProcessed&type%21=FileReference`;
+    const searchHref = `/search/?type=File&experiments.uuid=${encodeURIComponent(expUUID)}&type%21=FileProcessed&type%21=FileReference`;
 
     return (
         <div className="raw-files-table-section">
             <h3 className="tab-section-title">
                 <span><span className="text-400">{ files.length }</span> Raw File{ files.length === 1 ? '' : 's' }</span>
             </h3>
-            <FilesTableLoadedFromSearch {...{ width, schemas, columns, requestHref }} id={object.itemUtil.atId(context)} />
+            <EmbeddedItemSearchTable {...{ searchHref, schemas, columns, width }} facets={null} />
         </div>
     );
 });
 
 const ProcessedFilesTableSection = React.memo(function ProcessedFilesTableSection(props){
-    const { files, context } = props;
+    const { files, context, schemas } = props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['schemas' is assigned a value but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4660599)